src/jit/importer.cpp

```diff
@@ -12881,7 +12947,9 @@ MATH_MAYBE_CALL_NO_OVF:         ovfl = false;
 
             JITDUMP(" %08X", resolvedToken.token);
 
-            if (!opts.IsReadyToRun())
+            fNeedsRuntimeLookup = info.compCompHnd->computeNeedsRuntimeLookupAndHandleType(&resolvedToken, FALSE);
```

Comment:
Changed the architecture to have the impReadyToRunHelperToTree API return NULL if a runtime lookup is needed

---
In reply to: [62793738](https://github.com/dotnet/coreclr/pull/4519#discussion_r62793738) [](ancestors = 62793738)